control/statefbk.py

```diff
@@ -252,13 +253,13 @@ def acker(A, B, poles):
 
     # Place the poles using Ackermann's method
     n = np.size(p)
-    pmat = p[n-1]*a**0
+    pmat = p[n-1] * np.linalg.matrix_power(a, 0)
     for i in np.arange(1,n):
-        pmat = pmat + p[n-i-1]*a**i
+        pmat = pmat + np.dot(p[n-i-1], np.linalg.matrix_power(a, i))
```

Comment:
Again somewhat beside the point of this commit, but would it be possible to use Horner's method here?

Reply:
Added a TODO for someone to do this in a separate PR.